ts/sagas/profile.ts

```diff
@@ -178,15 +180,9 @@ function* createOrUpdateProfileSaga(
       throw new Error(response.right.value.title);
     }
 
-    if (response.right.status === 401) {
-      // on 401, expire the current session and restart the authentication flow
-      yield* put(sessionExpired());
-      throw new Error(I18n.t("profile.errors.upsert"));
-    }
-
     if (response.right.status !== 200) {
       // We got a error, send a SESSION_UPSERT_FAILURE action
-      throw new Error(response.right.value.title);
+      throw new Error(response.right.value?.title ?? "NO TITLE");
```

Comment:
Can we provide a better hint to someone/something receiving a "NO TITLE" error message? Maybe something like `${fileName}.${methodName}: received successful non-200 HTTP response with no "title" parameter in the body`

Reply:
What if we shorten to `Profile upsert: received successful non-200 HTTP response with no "title" parameter in the body`

Reply:
Fixed here https://github.com/pagopa/io-app/pull/4679/commits/e0f7fb63332cb23adc2cf487c8d94f5e64a4a6e4.